README.md

```diff
@@ -53,3 +53,54 @@ $ ./script/update -t arm64
 $ ./script/build -t arm64
 $ ./script/create-dist -t arm64
 ```
+
+## Releases
+
+There is no formal release process for libchromiumcontent, as release artifacts
+are created as a byproduct of CI. When a build is successful, its compiled
+asset is automatically uploaded to S3. These assets are later downloaded as 
+part of Electron's bootstrap script.  These files are about 4GB, so the 
+bootstrap task takes a while to run.
```

Comment:
If you run `npm run bootstrap -- --dev` it is much quicker and only downloads the shared libraries.